googleMapsApiModule.py

```diff
@@ -53,6 +55,5 @@ def geocoding(search_location):
     print("Latitude: "+str(result[0]['geometry']['location']['lat'])+" "+"Longitude: "+str(result[0]['geometry']['location']['lng']))
     address = search_location.lower()
     address = address.replace(" ","+")
-    url = "https://www.google.com/maps/search/?api=1&query="
-    result_url = url+address
+    result_url = "{}/{}".format(BASE_URL['geocoding'], address.lower().replace(" ", "+"))
```

Comment:
(Take necessary action, similar to above ^)

Reply:
> Looks like `__pycache__` files and some undesirable changes in `config.py` have crept into the last 2 commits. Can you remove those changes from this PR?

I don't know how to do that, please halp me out

Reply:
1. For `__pycache__` -> simply remove those files `rm -rf __pycache__`
2. For `config.py`:

Back-up your current working config file: `cp config.py config_backup.py`
Now, copy-paste the contents of config.py from master onto your local `config.py`

3. `git add config.py __pycache__`
4. `git commit ..."`
5. `git push ...`
6. Move your backup config back into place:

```sh
rm -rf config.py
mv config_backup.py config.py
```

***Most importantly, understand that `git add -A` or `git add .` is usually not a good practice to start with. Rather, manually add files like `git add a.txt b.txt` and then commit them. That way, you can be pretty sure what's being pushed to remote.***